js/blocks/components/edit.js

```diff
@@ -38,6 +38,7 @@ const Edit = ( { blockProps, block } ) => {
 						block={ `genesis-custom-blocks/${ block.name }` }
 						attributes={ attributes }
 						className="genesis-custom-blocks-editor__ssr"
+						httpMethod="POST"
```

Comment:
A [new](https://github.com/WordPress/gutenberg/tree/master/packages/server-side-render#httpmethod) `prop` for this component.